rllib/agents/dqn/dqn_policy.py

```diff
@@ -203,32 +202,21 @@ def build_q_model(policy, obs_space, action_space, config):
 
 
 def sample_action_from_q_network(policy, q_model, input_dict, obs_space,
-                                 action_space, config):
-    # Action Q network.
-    q_values, q_logits, q_dist = _compute_q_values(
-        policy, q_model, input_dict[SampleBatch.CUR_OBS], obs_space,
-        action_space)
-    policy.q_values = q_values
-    policy.q_func_vars = q_model.variables()
-
-    # Noise vars for Q network except for layer normalization vars
+                                 action_space, exploit, config, timestep):
+
+    policy.output_actions, policy.action_logp = \
+        simple_sample_action_from_q_network(
```

Comment:
Let's not do this indirection (I don't think it makes sense once we make the change to always call exploration action anyways).

Reply:
True, we could clean up DQN here ....